app/domain/authentication/authn_jwt/restriction_validation/validate_restriction_name.rb

```diff
@@ -0,0 +1,17 @@
+require 'command_class'
+
+module Authentication
+  module AuthnJwt
+    module RestrictionValidation
+      # Class to validate host annotation name is according the format of nested claim in JWT
+      class ValidateRestrictionName
```

Comment:
why is this not a command class?

Reply:
to be clear, i think this should be implemented as a method and not as a class at all. However, if it _is_ a class then it should be a command class

Reply:
We can discuss about turning this back to command class. But i don't understand the function solution.

ValidateGenericConfiguration - JWT class calls ValidateRestrictions
ValidateRestriction-  Is common to multiple authenticators call ExtractRestrictions 
ExtractRestriction - Common to multiple authenticators calls the extracting of annotation and turns them into restrictions. This is the place do the the validation . How a function in ValidateGenericConfiguration can fix this?